daemon/logger/jsonfilelog/jsonlog/jsonlogbytes.go

```diff
@@ -12,6 +12,7 @@ type JSONLogs struct {
 	Log     []byte    `json:"log,omitempty"`
 	Stream  string    `json:"stream,omitempty"`
 	Created time.Time `json:"time"`
+	Tag     string    `json:"tag,omitempty"`
```

Comment:
What's the reasoning behind making a new field instead of using `RawAttrs`?

Reply:
Mostly for clarity and (slightly?) easier parsing later.